www/shards/Portfile

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                shards
+version             0.8.1-devel
+
+github.setup        crystal-lang shards 0.8.1-devel v
+
+epoch               1
+categories          www
+platforms           darwin
+supported_archs     noarch
+license             Apache-2
+maintainers         {@conradwt gmail.com:conradwt}
```

Comment:
Can openmaintainer be added here?

Reply:
Yes, I can add `openmaintainer`.  Can you explain the reasoning for it?